mods/creative/init.lua

```diff
@@ -152,18 +185,18 @@ if minetest.setting_getbool("creative_mode") then
 			damage_groups = {fleshy = 10},
 		}
 	})
-	
+
 	minetest.register_on_placenode(function(pos, newnode, placer, oldnode, itemstack)
 		return true
 	end)
-	
+
 	function minetest.handle_node_drops(pos, drops, digger)
 		if not digger or not digger:is_player() then
 			return
 		end
 		local inv = digger:get_inventory()
 		if inv then
-			for _,item in ipairs(drops) do
+			for _, item in pairs(drops) do
```

Comment:
So in my case, `ipairs` is 5.5 times faster than `pairs`, and in HybridDog's case, `ipairs` is 3.8 times faster. My benchmark is also with LuaJIT, BTW.

But let's change it anyway because...reasons! (not specifically directed to you HybridDog)